src/Catel.Core/Catel.Core.Shared/Services/GuidObjectIdGenerator.cs

```diff
@@ -0,0 +1,25 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="GuidObjectIdGenerator.cs" company="Catel development team">
+//   Copyright (c) 2008 - 2018 Catel development team. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Catel.Services
+{
+    using System;
+
+    /// <summary>
+    /// Guid object generator
```

Comment:
Please use a . at the end of docs. It should read as a book, not like comments.

Reply:
Fixed